engine/src/main/java/org/terasology/recording/EventCopier.java

```diff
@@ -77,59 +48,59 @@
 
     }
 
-    Event copyEvent(Event e) {
-        if (e instanceof PlaySoundEvent) {
-            return e;
-        } else if (e instanceof BindButtonEvent) {
-            BindButtonEvent originalEvent = (BindButtonEvent) e;
-            BindButtonEvent newEvent = createNewBindButtonEvent(originalEvent);
+    Event copyEvent(Event toBeCopied) {
```

Comment:
Maybe an explicit access modifier? Good for consistency unless there is a very specific reason to be using default package private?